FluentUI.Demo/src/main/AndroidManifest.xml

```diff
@@ -2,28 +2,27 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     package="com.microsoft.fluentuidemo">
 
+    <uses-permission android:name="android.permission.INTERNET"/>
+
     <application
         android:allowBackup="true"
-        android:icon="@mipmap/ic_launcher"
+        android:icon="@drawable/fluent_logo"
```

Comment:
Quoting a [reference](https://stackoverflow.com/questions/28065267/mipmaps-vs-drawable-folders) on app/launcher icons -
It’s best practice to place your app icons in mipmap- folders (not the drawable- folders) because they are used at resolutions different from the device’s current density.

Reply:
Moved the icon to mipmap folder